apps/cli/package.json

```diff
@@ -9,10 +9,31 @@
     "@api7/adc-sdk": "workspace:*",
     "@types/express": "^5.0.3",
     "@types/supertest": "^6.0.3",
-    "supertest": "^7.1.4"
+    "supertest": "^7.1.4",
+    "vitest": "3.0.0",
+    "@nx/webpack": "21.6.4",
+    "webpack": "5.101.3",
+    "terser-webpack-plugin": "^5.3.14"
   },
   "dependencies": {
     "express": "^5.1.0",
-    "winston": "^3.17.0"
+    "winston": "^3.17.0",
+    "axios": "^1.13.1",
+    "rxjs": "^7.8.1",
+    "pluralize": "^8.0.0",
+    "listr2": "^8.2.1",
+    "commander": "^13.1.0",
+    "js-yaml": "^4.1.0",
+    "lodash": "^4.17.21",
+    "chalk": "^4.1.2",
+    "parse-duration": "^1.1.0",
+    "qs": "^6.12.1",
+    "dotenv": "^16.4.5",
+    "zod": "^4.0.10",
+    "source-map-support": "^0.5.21",
+    "agentkeepalive": "^4.6.0",
+    "glob": "^11.0.3",
+    "signale": "^1.4.0",
+    "@api7/adc-sdk": "workspace:*"
```

Comment:
It should be included in `devDependencies`.

https://nx.dev/docs/concepts/typescript-project-linking#set-up-package-manager-workspaces

Reply:
only `@api7/adc-sdk`